lib/kvs/kvs_rparams.c

```diff
@@ -307,12 +308,12 @@ static const struct param_spec pspecs[] = {
         .ps_stringify = param_default_stringify,
         .ps_jsonify = param_default_jsonify,
         .ps_default_value = {
-            .as_uscalar = 512 * 1024,
+            .as_uscalar = 512 << KB_SHIFT,
         },
         .ps_bounds = {
             .as_uscalar = {
-                .ps_min = 0,
-                .ps_max = UINT64_MAX,
+                .ps_min = 32 << KB_SHIFT,
+                .ps_max = 2 << MB_SHIFT,
```

Comment:
I would stick with just one shift unit (KB_SHIFT) as it makes size comparison to the reader a bit simpler.  But this is fine..

Reply:
I will keep this in mind for a future cleanup. I can see your point